source/MapPanel.cpp

```diff
@@ -458,25 +459,37 @@ void MapPanel::FinishDrawing(const string &buttonCondition)
 
 
 
-void MapPanel::DrawMiniMap(const PlayerInfo &player, float alpha, const System *const jump[2], int step)
+void MapPanel::DrawMiniMap(const PlayerInfo &player, float alpha, const System *const draw[2], int step)
```

Comment:
If the systems are going to be either both null, or the first one the same as currentSystem, does it still make sense to pass them both here? We could pass just the second one (front of the travel plan), and if that's null, go to the non-jump mode.

Reply:
Perhaps, but I'll note that this would result in a behavioral change. Currently, the minimap "remembers" the system you just jumped from until you've fully exited hyperspace and regained control of your ship in the next system, meaning both systems are shown throughout the duration of the jump. If all we did was pass in the front of the travel plan (unless Engine is still keeping track of it separately as it's effectively doing with the `minimapSystems` variable), then the minimap would update to instant your system changes.